docs/settings.rst

```diff
@@ -434,7 +446,7 @@ Overrides :ref:`setting-default-soa-edit`
 -  String
 
 .. deprecated:: 4.2.0
-  This setting has been deprecated and will be removed in 4.4.0
+  This setting has been removed in 4.4.0
```

Comment:
i was thinking it might be nicer if a new block would be added:
```
.. versionchanged:: 4.4.0
  This setting has been removed
```

so to keep the 4.2/4.3 docs "correct".

but maybe its clear enough.

Reply:
Both versions exist in the current documentation so I guess it is a matter of taste. I do not see how this this will make the 4.2/4.3 docs less "correct"